testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -142,17 +163,36 @@ private Process startContainer() throws IOException {
         builder.environment().put("KEYCLOAK_ADMIN_PASSWORD", "admin");
 
         if (restart.compareAndSet(false, true)) {
-            FileUtils.deleteDirectory(configuration.getProvidersPath().resolve("data").toFile());
+            if (SystemUtils.IS_OS_WINDOWS) {
+                deleteTempFilesOnWindows(configuration.getProvidersPath().resolve("data"));
+            } else {
+                FileUtils.deleteDirectory(configuration.getProvidersPath().resolve("data").toFile());
+            }
         }
 
         return builder.start();
     }
 
     private String[] getProcessCommands() {
         List<String> commands = new ArrayList<>();
-        commands.add(getCommand());
-        commands.add("-v");
+
+        if(SystemUtils.IS_OS_WINDOWS) {
+            commands.add(configuration.getProvidersPath().resolve("bin") + File.separator + SCRIPT_CMD_INVOKABLE);
+        } else {
+            commands.add(SCRIPT_CMD_INVOKABLE);
+            commands.add("-v"); // see #11185, has to be fixed before using -v on windows.
+        }
+
         commands.add("start");
+
+        if (SystemUtils.IS_OS_WINDOWS && firstStart.compareAndSet(true, false)) {
+            log.infof("===== FIRST START: Re-augmenting... =====");
+            log.infof("System: kc.home.dir: " + System.getProperty("kc.home.dir"));
```

Comment:
Note to self: remove this line when putting out of draft state. Its always null, failed debugging attempt.